### ISSUE TITLE
Order_lines / Invoice_lines

### ISSUE THREAD

**Issue by Jonas-johansen** [Tripletex/tripletex-api2#79]
Hi

When fetching invoices, should there not be a seperate invoice_line endpoint instead of reusing order_line?
Atleast when the order_lines for the invoice is not the same as on the order - it would make more sense for this to be called invoice_line..?

**Comment by ProCakeMan** [comment]
Agreed, the way it is right now makes little sense and just creates confusion.

**Comment by supertosse** [comment]
Hello
You make a valid point, but it's not easy to fix without breaking things. I don't see this being prioritized, but I will mention it to the team in case someone else disagree.
